planning/standards-status.md

```diff
@@ -0,0 +1,83 @@
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [vCard](https://datatracker.ietf.org/doc/html/rfc6350)             |  &#10004;      |                |                            |         |
+| [CardDAV](https://datatracker.ietf.org/doc/html/rfc4511)           |  &#10004;      |      N/A       |                            |         |
+
+## Calendar
+
+| Spec Name                                                          | Desktop Status | Android Status | Standards Involvement      | Comment |
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [iCalendar](https://datatracker.ietf.org/doc/html/rfc5545)         |  &#10004;      |                |                            |         |
+| [CalDAV](https://datatracker.ietf.org/doc/html/rfc4791)            |  &#10004;      |      N/A       |                            |         |
+| [CalDAV Scheduling](https://datatracker.ietf.org/doc/html/rfc6638) |                |      N/A       |                            |         |
+| [iTIP](https://www.rfc-editor.org/rfc/rfc2446)                     |  &#10004;      |                |                            |         |
+
+## Messaging
+
+| Spec Name                                                          | Desktop Status | Android Status | Standards Involvement      | Comment |
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [IRC](https://datatracker.ietf.org/doc/html/rfc1459)               |  &#10004;      |      N/A       |                            | IRC is actually over 2 RFCs: [RFC1459](https://datatracker.ietf.org/doc/html/rfc1459) and [RFC2812](https://datatracker.ietf.org/doc/html/rfc2812)  |
+| [IRC TLS Port](https://datatracker.ietf.org/doc/html/rfc7194)      |  &#10004;      |      N/A       |                            |         |
+| [IRC CTCP](http://www.alien.net.au/irc/ctcp.txt)                   |  &#10004;      |      N/A       |                            |         |
+| [IRC DCC](http://www.irchelp.org/irchelp/rfc/dccspec.html)         |  &#10004;      |      N/A       |                            |         |
+| [IRC SUPPORT](http://www.irc.org/tech_docs/005.html)               |  &#10004;      |      N/A       |                            | IRC SUPPORT is actually over 3 links: [ISUPPORT 00](https://datatracker.ietf.org/doc/html/draft-hardy-irc-isupport-00), [ISUPPORT 03](https://datatracker.ietf.org/doc/html/draft-brocklesby-irc-isupport-03), and [ISUPPORT 05](http://www.irc.org/tech_docs/005.html). (**TODO**: What about [ISUPPORT 01](https://datatracker.ietf.org/doc/html/draft-brocklesby-irc-isupport-01) and [ISUPPORT 02](https://datatracker.ietf.org/doc/html/draft-brocklesby-irc-isupport-02)?) |
```

Comment:
commenting this here since the idea came here, can we use footnote style links in gitbook? That might be a good way to represent this.

Reply:
what do you mean exactly? like a link of say `IRC SUPPORT` that would somehow direct folks to multiple links?

Reply:
Oh I see what you mean. I don't see why this wouldn't work in gitbook. In the case of multiple links, maybe the text like `IRC SUPPORT` wouldn't link to anything and instead have the 3 footnote links.. or we could leave the hyperlink in the text and still include the 3 footnotes. wdyt?

Reply:
Can you span multiple rows for a cell? That could let you list out the spec links separately in a separate column.

Reply:
yes we could. Although I'm a fan of moving all of the links to footnotes and I've verified that this will work in a table.

Reply:
I've moved all links into footnotes.